registration/include/pcl/registration/impl/ndt.hpp

```diff
@@ -76,6 +76,11 @@ NormalDistributionsTransform<PointSource, PointTarget, Scalar>::computeTransform
 {
   nr_iterations_ = 0;
   converged_ = false;
+  if (target_cells_.getCentroids()->empty()) {
+    PCL_ERROR("[%s::computeTransformation] Voxel grid is not searchable!\n",
```

Comment:
Maybe add why its not searchable?

Reply:
The other warning from the VoxelGridCovariance (`No voxels with a sufficient number of points. Grid will not be searchable. You can try reducing the min number of points required per voxel or increasing the voxel/leaf size`) will have been printed right before that. I think that's sufficient, what do you think?

Reply:
Ah, yes. Then its fine :+1: